### PR TITLE
Potentially Fixed IG Analytics [Needs Proper Testing]

### PR DESCRIPTION
**Pull request by ronakdev**
**Why**

Analytics were breaking IG 
**What Changed**

Describe the changes made to the code base including new features, refactoring of existing services and user visible API changes.

- moved analytics code to where the other javascript was being imported
- used older version of google analytics' tracking code that doesn't rely on an async external script import

**Choices Made**

**Tickets closed**:

**Future Work**
- I'm unfamilar with ig and I wasn't able to figure out where the generated html was being put — so I'm not 100% sure this will work
- I'm not sure what domain this will be on; if it is a different domain than `dpc.cms.gov`, we'll need to create a new thing in analytics (i can do this)

**Checklist**

- [ ] Demo and Seed commands are working
- [ ] Swagger documentation has been updated
- [ ] FHIR documentation has been updated

### CODE REVIEW

File: ig/source/resources/implementationguide/ig-new.json [hunk 1]
```diff
@@ -7,23 +7,23 @@
   },
   "extension" : [{
     "url" : "http://hl7.org/fhir/StructureDefinition/igpublisher-folder-resource",
-    "valueString" : "/Users/jaredcunha/work/dpc-app/ig/source/resources"
+    "valueString" : "/Users/ronakshah/Developer/HHS/dpc-app/ig/source/resources"
```
Comment (ronaldheft-usds) [inline ig/source/resources/implementationguide/ig-new.json]: @ronakdev @nickrobison-usds Are these files that should be added to .gitignore? They look like they have generated information in them?

Comment (ronakdev) [reply ig/source/resources/implementationguide/ig-new.json]: Good catch — maybe are those added when you run the `make` command?

Comment (nickrobison-usds) [reply ig/source/resources/implementationguide/ig-new.json]: They should be. I thought I git rm-ed them, but maybe not. @ronakdev Can you nuke these files before we merge? Thanks!

Comment (ronakdev) [reply ig/source/resources/implementationguide/ig-new.json]: Nuke the resources folder? Or remove the `valueString` property from `ig-new.json`?

Comment (ronaldheft-usds) [reply ig/source/resources/implementationguide/ig-new.json]: @ronakdev I assume remove the `ig-new.*` files from the Git history and add them to .gitignore.

**Pull request merged**